appholder/src/main/java/com/android/mdl/app/document/DocumentManager.kt

```diff
@@ -1247,7 +1247,7 @@ class DocumentManager private constructor(private val context: Context) {
         )
     }
 
-    fun showData(document: Document) {
+    fun showData(document: Document): CredentialDataResult.Entries? {
```

Comment:
We can remove this method as we moved it to TransferManager class

Reply:
Ah correct, we moved it there and this one got left behind. Will remove it